### ISSUE TITLE
Color contrast issue in the dashboard

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7193]
**CKAN version**
https://demo.ckan.org/dashboard/

**Describe the bug**

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (small)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Manage - mgifford - Users - CKAN Demo - https://demo.ckan.org/dashboard/

Element path: small

Snippet: <small>Activity from items that I'm following</small>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.47 (foreground color: #777777, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 107.0.1418.35

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by amercader** [comment]
This is the text that needs to be modified:
![Screenshot 2022-12-01 at 12-12-32 Manage - ckan_admin - Users - CKAN Demo](https://user-images.githubusercontent.com/200230/205038502-a0608f99-1705-42ff-bc63-be9d1c21a5d5.png)

**Comment by mgifford** [comment]
Yes. 
![Screenshot of grey text](https://user-images.githubusercontent.com/116832/208750646-690f7396-e217-4761-a446-9a0dd0701ffc.png)
